lib/agent0/agent0/hyperdrive/exec/run_agents.py

```diff
@@ -380,3 +391,99 @@ def build_wallet_positions_from_data(
         longs=long_obj,
         shorts=short_obj,
     )
+
+
+def build_wallet_positions_from_chain(
```

Comment:
I think we should put this in a file with `*_from_data`. so `build_wallet_positions.py`. We've been tending in that direction, and it's obvious where the test is (`build_wallet_positions_test.py`).

Reply:
this is hard to unit test, I need the system as a whole to test wallet states, hence the test is in the outer directory. I can move these to a separate file.

Reply:
sure we can punt on tests for now, just mentioning that as a future goal.